Logic.json

```diff
@@ -74,7 +75,7 @@
 		"Terrydactyland_Early": [["Relic", "Clockworks", "Egg_Aim", "Glitches"]],
 		"Nest": [["Terrydactyland", "Springy_Step_Shoes"], ["Terrydactyland", "Clockwork_Warp"], ["Terrydactyland", "Split_Up", "Clockworks", "Egg_Aim", "Glitches"]],
 		"Oogle_Cave_K": [["Oogle_Cave_BK", "Split_Up"], ["Terrydactyland", "Split_Up", "SuperBanjo", "Glitches"]],
-		"Oogle_Cave_BK": [["Terrydactyland", "Clockwork_Warp"], ["Terrydactyland", "Dinosaur", "TDL_Mumbo"], ["Terrydactyland", "Nest", "Glitches"]],
+		"Oogle_Cave_BK": [["Terrydactyland", "Clockwork_Warp"], ["Terrydactyland", "Dinosaur", "TDL_Mumbo"], ["Terrydactyland", "Nest", "Glitches"], ["Terrydactyland", "Split_Up", "SuperBanjo", "Claw_Clambers", "Glitches"], ["Terrydactyland", "Split_Up", "SuperBanjo", "Tricks", "Glitches"]],

```

Comment:
Can make this cleaner by using Oogle_Cave_K. If you are trying to avoid circular references, its not a problem here